### DISCUSSION TITLE
Add descriptive page titles on nicegui.io, e.g. for more informative bookmarking

### DISCUSSION THREAD

**Discussion by bandit-masked** [zauberzeug/nicegui#2583] [Ideas / Feature Requests]
If you bookmark any page on nicegui.io, the description is always "NiceGUI". It's because the title of each page simply reads "NiceGUI". If you are bookmarking various sections, you end up with a long list of bookmarks all named "NiceGUI", which makes it more difficult to find that one relevant page.

It seems more beneficial if the page titles were more descriptive, e.g. "NiceGUI | Table" or "NiceGUI | Text elements". At the moment, the URL is more informative than the page title.

**Comment by falkoschindler** [comment]
Great idea, @bandit-masked!
I just created PR https://github.com/zauberzeug/nicegui/pull/2607 to improve these page titles. 🙂

**Reply by bandit-masked** [reply]
Thank was quick! 👍 Thanks!

**Comment by falkoschindler** [comment]
This feature has just been released in version [1.4.16](https://github.com/zauberzeug/nicegui/releases/tag/v1.4.16) and will be online shortly. 🎉
